v2/internal/frontend/desktop/windows/win32/window.go

```diff
@@ -37,6 +38,16 @@ func IsWindowMaximised(hwnd uintptr) bool {
 	return style&WS_MAXIMIZE != 0
 }
 
```

Comment:
I think we may be missing `IsWindowFullscreen()`?

Reply:
I first used the form function isfullscreen.
But I have added the native implementation.